pkg/protoc/wrapper.go

```diff
@@ -116,3 +122,74 @@ func (e *wrapperImpl) execProtoc(ctx context.Context, protoPath string) error {
 
 	return nil
 }
+
+func (e *wrapperImpl) commands(ctx context.Context, protoPath string) ([][]string, error) {
+	cmds := make([][]string, 0, len(e.cfg.Plugins))
+	relProtoPath, _ := filepath.Rel(e.rootDir.String(), protoPath)
+
+	outDir, err := e.cfg.OutDirOf(e.rootDir.String(), protoPath)
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+
+	isMod := e.modulesEnabled()
+
+	funcMap := template.FuncMap{
+		"module": func(in string) (string, error) {
+			if isMod {
+				return e.getModulePath(ctx, in)
```

Comment:
**[wraperr]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
	pkg/protoc/wrapper.go:140:12:	`return e.getModulePath(ctx, in)`